doc/masternode_conf.md

```diff
@@ -27,3 +27,23 @@ The following RPC commands are available (type `help masternode` in Console for
 * start-missing
 * start-disabled
 * outputs
+-----
+Edit the ravendark.conf file on the vps with at least the following details:\
+daemon=1\
+masternode=1\
+masternodeprivkey=masternode private key\
+rpcallowip=127.0.0.1\
+rpcuser=\
+rpcpassword=\
```

Comment:
`bind=0.0.0.0` can be important also, if not, can't hurt to have it in there.

Reply:
Not necessary, so I didnt include it.